content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/critical-cve-policy.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Ensure there are no High or Critical CVEs in your workloads
+"
+type: "article"
+description: "Understanding Identity and Access Management in Chainguard Enforce"
+date: 2023-04-12T15:22:20+01:00
+lastmod: 2023-04-12T15:22:20+01:00
+draft: false
+images: []
+tags: ["policy-controller", "Policies", "Enforce", "Product"]
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 820
+toc: true
+---
+
+While Common Vulnerabilities and Exposures (CVEs) are undesirable at any time, the security standards of some industries strictly regulate the allowance of _high_ or _critical_ CVEs in industry-specific software.  For example, in the payment industry, the [PCI Security Standards Council](https://www.pcisecuritystandards.org/) requires that all vulnerabilities with a Common Vulnerability Scoring System (CVSS) score higher than 4 are addressed.
```

Comment:
What do we mean by the vulnerabilities "are addressed"?

```suggestion
While Common Vulnerabilities and Exposures (CVEs) are undesirable at any time, the software security standards of certain industries strictly regulate the allowance of _high_ or _critical_ CVEs.  For example, in the payment industry, the [PCI Security Standards Council](https://www.pcisecuritystandards.org/) requires that all vulnerabilities with a Common Vulnerability Scoring System (CVSS) score higher than 4 are addressed.
```

Reply:
That was language Matt Moore used elsewhere. I think it refers to the fact that these CVEs are not entirely prohibited, they just have to be investigated to make sure they don't pose a real risk. 

Would switching "addressed" to "resolved" work? I'm not sure I want to get too off topic by explaining the above, but I also don't want to confuse readers!

Reply:
"Addressed" is fine but it seems like it's missing something, like a time horizon or something